cosmic-core/api/src/main/java/com/cloud/api/response/WhoHasThisIpResponse.java

```diff
@@ -0,0 +1,150 @@
+package com.cloud.api.response;
+
+import com.cloud.api.ApiConstants;
+import com.cloud.api.BaseResponse;
+import com.cloud.network.IpAddress;
+import com.cloud.network.Networks;
+import com.cloud.serializer.Param;
+import com.cloud.utils.net.Ip;
+import com.cloud.vm.Nic;
+
+import java.net.URI;
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+
+public class WhoHasThisIpResponse extends BaseResponse {
+
+    @SerializedName(ApiConstants.VPC_NAME)
+    @Param(description = "VPC name")
+    private String vpcname;
+
+    @SerializedName(ApiConstants.IP_ADDRESS)
+    @Param(description = "IP address")
+    private String ipaddress;
+
+    @SerializedName(ApiConstants.NETWORK_NAME)
+    @Param(description = "Network name")
+    private String networkname;
+
+    @SerializedName(ApiConstants.ASSOCIATED_NETWORK_NAME)
+    @Param(description = "Network name")
+    private String associatedNetworkName;
+
+    @SerializedName(ApiConstants.PRIVATE_MAC_ADDRESS)
```

Comment:
You created `ApiConstants.NICS_MAC_ADDRESS` but are using `ApiConstants.PRIVATE_MAC_ADDRESS`. An autocompletion mistake probably?

NOTE: Why not just `ApiConstants.MAC_ADDRESS`?